internal/testtypes/builders.go

```diff
@@ -62,6 +66,30 @@ func (c *ClusterBuilder) WithClusterClass(class clusterv1.ClusterClass) *Cluster
 	return c
 }
 
+// WithTopologyVersion adds the passed TopologyVersion to the ClusterBuilder.
+func (c *ClusterBuilder) WithTopologyVersion(topologyVersion string) *ClusterBuilder {
```

Comment:
Q: given that Topology is a nested struct behind Cluster, is it possible to have a something that reads Has `WithTopology` and gets in input a Topology builder instead of having several top level methods?

Reply:
Good idea - I'll refactor this.